ipinfo/bogon.py

```diff
@@ -0,0 +1,66 @@
+from ipaddress import ip_network, ip_address as IP
+
+
+def is_bogon(ip_address):
+    for network in BOGON_NETWORKS:
+        if IP(ip_address) in ip_network(network):
```

Comment:
Can we pre-convert all of this to the object type instead of converting for each loop iteration? We do the same in the Java SDK.